src/gatsby-node.js

```diff
@@ -20,7 +20,7 @@ export const onCreateWebpackConfig = (
 };
 
 /**
- * Moves all js and css files into timestamp-named folder
+ * Moves assets to assets folder
```

Comment:
not sure how this comment is an improvement.

Reply:
It could probably be improved, yes :) Any suggestions? My reason for changing it was that the current one is incorrect since it says that the files are moved to a `timestamp-named` folder. There is no timestamp added here...